tools/server/webui/src/components/ChatScreen.tsx

```diff
@@ -132,7 +128,7 @@ export default function ChatScreen() {
     scrollToBottom(true);
   };
 
-  const sendNewMessage = async () => {
+  const sendNewMessage = useCallback(async () => {
```

Comment:
no need `useCallback` here. We can simply update `textarea.onSubmit = sendNewMessage` each time